cpp/include/legate_dataframe/core/column.hpp

```diff
@@ -376,6 +477,7 @@ class PhysicalColumn {
   const cudf::data_type cudf_type_;
   mutable std::vector<std::unique_ptr<cudf::column>> tmp_cols_;
   mutable std::vector<rmm::device_buffer> tmp_null_masks_;
+  const bool scalar_out_;  // scalar output checks binding has size 1.
```

Comment:
Opted to call this differently, because it seemed to matter less here and I ended up only using it to check if binding a result adds up.